src/django-react-loader.js

```diff
@@ -1,3 +1,5 @@
+const path = require('path')
+
```

Comment:
Does `import * as path from 'path'` work, or does it have to be via `require` to work?